.travis.yml

```diff
@@ -1,16 +1,55 @@
-language: python
+matrix:
+  include:
+    - language: python
+      os:
+        - linux
+        # Travis does not support python on osx yet.
+        # see: https://github.com/travis-ci/travis-ci/issues/2312
+        # - osx
```

Comment:
Really? We should look into using azure-pipelines... [It's working well in deno_std](https://github.com/denoland/deno_std/blob/master/azure-pipelines.yml)

Reply:
Well, but this shouldn't block this PR, right?

I think we should move to azure-pipelines if this repo is merged into deno_std.